### DISCUSSION TITLE
Soulmask

### DISCUSSION THREAD

**Discussion by Nemesis851** [gameserverapp/Platform#832] [Game request]
I played the Beta and it has a LARGE map, many tribes, has both PVE and PVP servers.
Many video on YouTube covering it

Here is the Steam Store page
**https://store.steampowered.com/app/2646460/Soulmask/**

Estimated release is Q2 2024

**Comment by ad3d0** [comment]
this one is out in EA now, pretty positive reviews so far.

**Comment by birdman3131** [comment]
https://soulmask.fandom.com/wiki/Private_Server some info on private servers

**Comment by Eurymachus** [comment]
Played this today and it's lots of fun! Would definitely love to have this managed by my GSA <3

**Comment by mvaessen** [comment]
For the people that are looking for a Soulmask blueprint, I posted them on Discord. Both Windows and Linux blueprints are available.

**Reply by Eurymachus** [reply]
Awesome work! Just trying the Windows one now <3

Stuck at 5% though, gonna leave it cook till morning :D

**Comment by mvaessen** [comment]
This game is now officially supported
